src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirCosmosResourceWrapper.cs

```diff
@@ -105,5 +113,52 @@ internal string GetETagOrVersion()
 
             return base.Version;
         }
+
+        public override void UpdateSearchIndices(IReadOnlyCollection<SearchIndexEntry> searchIndices)
+        {
+            base.UpdateSearchIndices(searchIndices);
+
+            UpdateSortIndex(searchIndices);
+        }
+
+        private void UpdateSortIndex(IReadOnlyCollection<SearchIndexEntry> searchIndices)
+        {
+            SortValues = new Dictionary<string, SortValue>();
+
+            if (searchIndices == null)
+            {
+                return;
+            }
+
+            foreach (SearchIndexEntry entry in searchIndices)
+            {
+                if (entry.Value is ISupportSortSearchValue sortValue && (sortValue.IsMax || sortValue.IsMin))
+                {
+                    if (SortValues.TryGetValue(entry.SearchParameter.Code, out SortValue item))
+                    {
+                        SetSortValue(item, sortValue, entry);
+                    }
+                    else
+                    {
+                        var sortValueEntry = new SortValue(entry.SearchParameter.Url);
+                        SetSortValue(sortValueEntry, sortValue, entry);
+                        SortValues[entry.SearchParameter.Code] = sortValueEntry;
+                    }
+                }
+            }
+
+            void SetSortValue(SortValue item, ISupportSortSearchValue sortValue, SearchIndexEntry entry)
+            {
+                if (sortValue.IsMax)
+                {
+                    item.High = entry.Value;
+                }
+
+                if (sortValue.IsMin)
+                {
+                    item.Low = entry.Value;
+                }
```

Comment:
super nit:
There is check before we call this method `(sortValue.IsMax || sortValue.IsMin)` so I I would went with simple `if else` statement rather than two `if`